twitter/API/APIManager.m

```diff
@@ -47,29 +51,20 @@ - (instancetype)init {
     return self;
 }
 
-- (void)getHomeTimelineWithCompletion:(void(^)(NSArray *tweets, NSError *error))completion {
-    
-    [self GET:@"1.1/statuses/home_timeline.json"
-   parameters:nil progress:nil success:^(NSURLSessionDataTask * _Nonnull task, NSArray *  _Nullable tweetDictionaries) {
-       
-       // Manually cache the tweets. If the request fails, restore from cache if possible.
-       NSData *data = [NSKeyedArchiver archivedDataWithRootObject:tweetDictionaries];
-       [[NSUserDefaults standardUserDefaults] setValue:data forKey:@"hometimeline_tweets"];
-
-       completion(tweetDictionaries, nil);
+- (void)getHomeTimelineWithCompletion:(void(^)(NSMutableArray *tweets, NSError *error))completion
```

Comment:
Probably shouldn't return mutable array.

Reply:
Why?

Reply:
I think it takes up more memory. But yeah. Maybe it’s not so weird. I just don’t think I ever see it done so I assumed there is a good reason. Maybe it’s fine.